src/App/Routes/Modals/RegisterModal/GithubSignin/View.tsx

```diff
@@ -46,17 +48,23 @@ const GithubSignin: FC<PropsWithChildren<Props>> = ({
       className={className}
     >
       {children || <Button as="div">Connect to Github</Button>}
-      {renderLoadingModal()}
+      {renderLoadingModal(txHash)}
       {renderError()}
     </GithubConnect>
   );
 
-  function renderLoadingModal() {
+  function renderLoadingModal(txHash?: string) {
     return (
       <Modal contentClassName="px-16 pt-6 pb-8 z-100" isOpen={isLoading} onClose={onClose}>
         <div className="flex flex-col justify-center items-center text-4xl text-center text-blue-500 mt-12">
           <Loader className="animate-spin mr-4 mb-4" size={62} />
           <div>We are verifying your information</div>
+          {txHash && (
```

Comment:
You don't need this as there is no transaction ongiong.
Just change the message and close the modal when the user signed (or reject) the message